### ISSUE TITLE
Brand footer SVG links in the examples are missing appropriate link name

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#448]
# Bug Report

## Description
We want to make sure links have discernible text. In our brand footer examples, we have a link that wraps an SVG, but no text is provided for the link. This reads as an empty link element.

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-ui/#/components/terra-brand-footer/brand-footer/brand-footer
2. Run [aXe Chrome extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)

## Expected Behavior
Links have discernible text

## Possible Solution
Look into ways to add discernible text to SVG or use terra-visually-hidden-text component in addition to SVG

## Environment

* Component Name and Version: terra-brand-footer
